### DISCUSSION TITLE
Internationalization

### DISCUSSION THREAD

**Discussion by Garcia-INPE** [cu-esiil-edu/2024-data-short-course#29] [Q&A]
I'm wondering if there is a manner to internationalize GitHub Pages, i.e., interchange pt-BR or en-US on-the-fly, any idea folks? Thanks.

**Comment by oye-bobs** [comment]
check out this article @Garcia-INPE

**Reply by Garcia-INPE** [reply]
Which article?

**Reply by oye-bobs** [reply]
Oh I'm sorry I thought I added the link.
@Garcia-INPE 

Here it is : https://mikemcquaid.com/translations-with-rails-and-jekyll/

**Comment by eculler** [comment]
@Garcia-INPE I think it depends on the template you use, but the one that I use for the syllabus allows me to set the language: https://github.com/mmistakes/jekyll-theme-basically-basic/blob/master/_config.yml

I imagine there are some themes that have better support for internationalization, like this one: https://github.com/letrastudio/jekyll-i18n

It also looks like there are plug-ins out there, but I would imagine that's a fair bit more work to get the site up and running
